admin/admin-base.php

```diff
@@ -23,7 +23,7 @@ public function __construct( &$links_model ) {
 		parent::__construct( $links_model );
 
 		// Plugin i18n, only needed for backend
-		load_plugin_textdomain( 'polylang', false, basename( POLYLANG_DIR ) . '/languages' );
+		load_plugin_textdomain( 'polylang' );
```

Comment:
I think we do not need this function on Polylang thanks to "Requires at least: 4.9"